### PR TITLE
Release v17.2.0

### PR DESCRIPTION
**Pull request by shekhirin**

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ethabi-cli"
-version = "17.1.0"
+version = "17.2.0"
```
Comment (nlordell) [inline cli/Cargo.toml:3]: Just curious, why not a minor version bump (since its just a bug fix for nested tuple and array parsing)?

Comment (shekhirin) [reply cli/Cargo.toml:3]: thought that parsing nested arrays/tuples is a new feature rather than a bugfix, but can change to patch bump if you want to

Comment (nlordell) [reply cli/Cargo.toml:3]: I'm not so familiar with the `ethabi` CLI tool, so I was just curious about the reasoning, not necessarily suggesting it should be one way or the other.

**Review by nlordell — APPROVED** [review]

**Review by vkgnosis — APPROVED** [review]

**Comment by vkgnosis** [comment]
I'll make the crates.io release.

**Comment by vkgnosis** [comment]
Turns out nlordell and I are only owners of the main ethabi crate and not the other crates (ethabi-cli, ethabi-contract, ethabi-derive) so we can't release those.

**Comment by vkgnosis** [comment]
Since it's a binary crate maybe ethabi-cli doesn't need to be on crates.io anyway? You could use it from git.

**Pull request merged**